assets/js/card.js

```diff
@@ -0,0 +1,12 @@
+window.onload = function () {
```

Comment:
presumably they use this card component elsewhere and  it requires this jquery?

Reply:
yes. Its a way of making the whole tile clickable, and for the click to follow the link defined in the card header.

Reply:
Actually, just checking that, it doesn't. Either it doesn't do that or the script is not active. I'll dig a bit more in next PR.